server/autotest_server/__init__.py

```diff
@@ -97,12 +104,16 @@ def _create_test_script_command(tester_type: str) -> str:
     Return string representing a command line command to
     run tests.
     """
-    import_line = f"from testers.{tester_type}.{tester_type}_tester import {tester_type.capitalize()}Tester as Tester"
+    resource_settings = get_resource_settings(config)
+    import_line = (
+        f"from testers.{tester_type}.{tester_type}_tester" f" import {tester_type.capitalize()}Tester as Tester"
+    )
     python_lines = [
-        "import sys, json",
+        "import sys, json, resource",
         f'sys.path.append("{os.path.dirname(os.path.abspath(__file__))}")',
         import_line,
         "from testers.specs import TestSpecs",
+        *get_setrlimit_lines(resource_settings),
```

Comment:
More generally, I'd rather not expand this ad-hoc Python code. Instead of generating the commands here, please pass the raw resource settings (i.e., the `list[tuple[int, tuple[int, int]]]`) as a new argument to the `Tester` initializer and have the `Tester` set the resource limits.